sync_test.go

```diff
@@ -356,21 +356,11 @@ func TestSetup(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error geting namespace: %s\n", err)
 	}
-	config := neutrino.Config{
-		DataDir:     tempDir,
-		Database:    db,
-		ChainParams: modParams,
-		AddPeers: []string{
-			h3.P2PAddress(),
-			h2.P2PAddress(),
-			h1.P2PAddress(),
-		},
-	}
 
 	neutrino.MaxPeers = 3
 	neutrino.BanDuration = 5 * time.Second
 	neutrino.WaitForMoreCFHeaders = time.Second
-	svc, err := neutrino.NewChainService(config)
+	svc, err := neutrino.NewChainService(neutrino.DataDir(tempDir), neutrino.Database(db), neutrino.ChainParams(modParams), neutrino.AddPeers([]string{ h3.P2PAddress(), h2.P2PAddress(),h1.P2PAddress()}))
```

Comment:
Style nit: we typically attempt to wrap all lines to 80 columns on a best effort basis.